README.md

```diff
@@ -590,11 +608,14 @@ appropriate values, or falls back on its own defaults.
 
 The `config` command accepts the following subcommands:
 
--   `show` prints the current or specified version configuration
--   `write` store the current or specified version configuration
--   `edit` opens the current or specified version configuration in an editor
-    (Using $EDITOR, e.g: export `EDITOR=/usr/bin/vim`)
--   `delete` removes the specified configuration
+
+- `show` prints the current or specified version configuration
```

Comment:
Markdown standard it four chars wide; I've seen some Markdown parsers fail to parse this format as a list (not on GH tho). That's why I had indented them.

Reply:
fixed in 82cf4d5